tests/Doctrine/Tests_PHP74/Persistence/Mapping/RuntimeReflectionServiceTest.php

```diff
@@ -45,6 +55,33 @@ public function testGetTypedPublicNoDefaultPropertyWorksWithGetValue(): void
     {
         $reflProp = $this->reflectionService->getAccessibleProperty(self::class, 'typedNoDefaultPublicProperty');
         self::assertInstanceOf(RuntimePublicReflectionProperty::class, $reflProp);
+        self::assertInstanceOf(TypedNoDefaultRuntimePublicReflectionProperty::class, $reflProp);
```

Comment:
Original assertion can be dropped
```suggestion
        self::assertInstanceOf(TypedNoDefaultRuntimePublicReflectionProperty::class, $reflProp);
```

Reply:
It was kept on purpose, to ensure that there is no "BC break"

Reply:
I see. Feel free to leave it in then 👍